packages/template-tag-codemod/src/index.ts

```diff
@@ -105,7 +110,11 @@ export async function ensureAppSetup() {
     process.exit(-1);
   }
   if (!pkg.packageJSON.exports) {
-    throw new Error(`must use package.json exports for self-resolvability. Plase add this to package.json:
+    pkg.packageJSON.exports = {
```

Comment:
This is mutating the package cache. It might happen to work, but it changes the meaning of the package cache from an optimization to a correctness requirement.

I'm in favor of auto-fixing this problem for users, but we should do it by writing their real package.json. Not trying to maintain an inconsistency in the package cache.